cost-analyzer/values.yaml

```diff
@@ -181,6 +181,7 @@ kubecost:
 kubecostModel:
   image: "gcr.io/kubecost1/cost-model"
   imagePullPolicy: Always
+  outOfClusterPromMetricsEnabled: false
```

Comment:
Quick doc or comment on what this means?

Reply:
Done.